### DISCUSSION TITLE
Does ESP32-wroom-32E support Micropython?

### DISCUSSION THREAD

**Discussion by zhy2020** [micropython/micropython#10687] [General]
Does ESP32-wroom-32E support Micropython?

**Comment by davefes** [comment]
> The two modules are pin-to-pin compatible. The main difference is that ESP32-WROOM-32UE integrates ESP32 ECO V3 chip, and has enhanced security boot and flash encryption.

I have a couple of ESP32-WROOM-32 with rev 3 chips in them and they seem to behave the same as rev 1 in my applications.

**Reply by zhy2020** [reply]
thanks
